hangman/game.py

```diff
@@ -1,23 +1,76 @@
 from .exceptions import *
+from random import randint
 
 # Complete with your own, just for fun :)
-LIST_OF_WORDS = []
+LIST_OF_WORDS = ['Python', 'Hello World', 'Women Who Code']
 
 
 def _get_random_word(list_of_words):
     pass
+    if len(list_of_words) != 0:
+       random_index = randint(0, len(list_of_words)-1)
+    else:
+        raise InvalidListOfWordsException()
+    return list_of_words[random_index];
 
 
 def _mask_word(word):
     pass
+    if word != '':
+        letters_of_word = list(word)
```

Comment:
you don't need to convert string to list .. in python, string is iterable as well.. meaning you can do this..

```python
for letter in word:
     print(letter)
```